kinto/core/storage/postgresql/__init__.py

```diff
@@ -533,9 +533,19 @@ def purge_deleted(self, collection_id, parent_id, before=None,
             placeholders['before'] = before
 
         with self.client.connect() as conn:
-            result = conn.execute(query.format_map(safeholders), placeholders)
+            result = conn.execute(delete_tombstones.format_map(safeholders), placeholders)
+            deleted = result.rowcount
+
+            # If purging everything from a parent_id, then clear timestamps.
+            if collection_id is None and before is None:
```

Comment:
Indeed, the tombstones are the last bit of info that we keep about a resource. Once we delete the tombstones, we can wipe out timestamps.

I agree that changing the name would be the best option.